src/main/java/com/example/bookwonders/dto/cart/AddCartItemRequestDto.java

```diff
@@ -0,0 +1,15 @@
+package com.example.bookwonders.dto.cart;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Positive;
+import lombok.Data;
+
+@Data
+public class AddCartItemRequestDto {
+    @NotBlank
+    @Positive
+    private Long bookId;
+    @NotBlank
+    @Positive
+    private int quantity;
```

Comment:
```suggestion
    @NotNull
    @Positive
    private Integer quantity;
```
why not to uniform bookId and quantity